README.md

```diff
@@ -1,191 +1,6 @@
-
-Default log level is set to `warning`. Log level can be increased during container deployment passing the `-e QGIS_SERVER_LOG_LEVEL=0` option:
-
-```sh
-docker run -e QGIS_SERVER_LOG_LEVEL=0 -v $(pwd)/data:/io/data -v $(pwd)/plugins:/io/plugins --name qgis-server -d -p 8010:80 qgis/qgis-server:ltr
-```
-
-### Run multiple instances of QGIS server
-
-Adjust first the configuration in `conf/nginx.conf` with the proper number of expected workers
-and `docker-compose.yml` with the path of data folder on the host.
-
-Then:
-
-```sh
-docker-compose up -d --scale qgis-server=N
-```
-
-Where N is the number of expected QGIS server workers.
-
-
-### Data dir structure
-
-`$(pwd)/data` must have the following structure:
-
-```text
-data
- |
- |-- <project_name>
-      |-- <project_name>.qgs
-```
-
-[oq-consolidate](https://github.com/gem/oq-consolidate) may helps you in exporting data suitable for QGIS server (consolidating project and layers).
-
-`$(pwd)/plugins` must have the following structure:
-
-```text
-plugins
- |
- |-- <plugin_name>
-      |-- <plugin_code>.py
-      |-- metadata.txt
-      |-- __init__.py
-```
-
-Custom fonts are loaded into `/usr/share/fonts`. `fc-cache` is run when container is started.
-
-### Runtime customizations
-
-The following variables can be customized during container deployment:
-
-#### Nginx
-
-When `SKIP_NGINX` is set to a different value than `0` or `false` the embedded copy of Nginx will not be started and an external reverse proxy is then required to access the FastCGI QGIS backend.
-
-- `SKIP_NGINX`: default is *unset* (do not skip Nginx startup)
-
-#### QGIS
-
-- `QGIS_SERVER_LOG_LEVEL`: default is `1`
-- `QGIS_SERVER_PARALLEL_RENDERING`: default is `true`
-- `QGIS_SERVER_MAX_THREADS`: default is `2`
-- `QGIS_SERVER_WMS_MAX_WIDTH`: not set by default
-- `QGIS_SERVER_WMS_MAX_WIDTH`: not set by default
-
-See [QGIS server documentation](https://docs.qgis.org/testing/en/docs/server_manual/config.html#environment-variables) for further details.
-
-It is also possible to customized the ID of the user running QGIS server processes when it is required to write to host volumes (see [notes](#notes)):
-
-- `QGIS_USER`: default is `nginx`, a numerical id must be proivided
-
-Example: `docker run -e QGIS_USER=1000` or `docker run -e QGIS_USER=$(id -u qgis)`
-
-### Notes
-
-GeoPackages do not play well with multiple processes having gpkg files opened in `rw` mode. By default QGIS server processes lack write permission on `/io/data`.
-If it is required to let QGIS server write data to `/io/data` make sure that either you are using a process-safe datastore (i.e. a Postgres backend) or you are
-limiting horizontal scaling to one container only. QGIS server user can be customized via the `QGIS_USER` environment variable.
-
-## QGIS Desktop
+### Warning
 
-There is currently no documentation how to run the QGIS Desktop via docker.
+There are discussions on how to build these images and they are not considered stable. 
```

Comment:
is there any link to this discussions?

Reply:
no, no formal discussions yet.
The main discussion in the past was about building QGIS rather than using apt-get install.